terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/main/java/org/terasoluna/gfw/common/codelist/i18n/SimpleI18nCodeList.java

```diff
@@ -226,14 +232,54 @@
 
     /**
      * <p>
-     * returns row of codelist table.
+     * Returns a codelist as map for the {@link org.springframework.context.i18n.LocaleContextHolder}'s locale.
+     * </p>
+     * @see org.terasoluna.gfw.common.codelist.CodeList#asMap()
+     */
+    @Override
+    public Map<String, String> asMap() {
+        return asMap(LocaleContextHolder.getLocale());
+    }
```

Comment:
- [ ] I think that you should change `AbstractI18nCodeList#asMap()`.

Reply:
Modified by cddccf8